test/unit/helpers/xhr/MFA_VERIFY_invalid_password.js

```diff
@@ -0,0 +1,16 @@
+define({
+  'status': 403,
+  'responseType': 'json',
+  'response': {
+    'errorCode': 'E0000068',
+    'errorSummary': 'Invalid Passcode\/Answer',
```

Comment:
Is this correct? I would think the server will return ```Invalid Password``` or something along those lines

Reply:
Yea, it's correct. The error summary is `Invalid Passcode/Answer` while the errorCause is `Password is incorrect`.